static/app/views/settings/organizationIntegrations/docIntegrationDetailedView.tsx

```diff
@@ -95,16 +74,58 @@ class DocIntegrationDetailedView extends AbstractIntegrationDetailedView<
         </LearnMoreButton>
       </ExternalLink>
     );
-  }
+  }, [doc, integrationSlug, integrationType, installationStatus, organization]);
 
-  renderIntegrationIcon() {
-    return <DocIntegrationAvatar docIntegration={this.integration} size={50} />;
+  if (isPending) {
+    return <LoadingIndicator />;
   }
 
-  // No configurations.
-  renderConfigurations() {
-    return null;
+  if (!doc) {
+    return <LoadingError message={t('There was an error loading this integration.')} />;
   }
+
+  return (
+    <IntegrationLayout.Body
+      integrationName={integrationName}
+      alert={null}
+      topSection={
+        <IntegrationLayout.TopSection
+          featureData={featureData}
+          integrationName={integrationName}
+          installationStatus={installationStatus}
+          integrationIcon={<DocIntegrationAvatar docIntegration={doc} size={50} />}
+          addInstallButton={
+            <IntegrationLayout.AddInstallButton
+              featureData={featureData}
+              hideButtonIfDisabled={false}
+              requiresAccess={false}
+              renderTopButton={renderTopButton}
+            />
+          }
+          additionalCTA={null}
+        />
+      }
+      tabs={
+        <IntegrationLayout.Tabs
+          tabs={tabs}
+          activeTab={'overview'}
+          onTabChange={() => {}}
+          getTabDisplay={(tab: Tab) => tab}
+        />
+      }
+      content={
+        <IntegrationLayout.InformationCard
+          integrationSlug={integrationSlug}
+          description={description}
+          alerts={[]}
```

Comment:
Passing an empty array here (or the noop function above) will use a new reference on each rerender. Probably doesn't matter but generally it's best to declare them outside the component so it's a stable reference